samcli/lib/providers/sam_function_provider.py

```diff
@@ -160,20 +160,30 @@ def _convert_sam_function_resource(name, resource_properties, layers, ignore_cod
         samcli.commands.local.lib.provider.Function
             Function configuration
         """
-
         codeuri = SamFunctionProvider.DEFAULT_CODEURI
+        inlinecode = None
         imageuri = None
         packagetype = resource_properties.get("PackageType", ZIP)
         if packagetype == ZIP:
-            codeuri = SamFunctionProvider._extract_sam_function_codeuri(
-                name, resource_properties, "CodeUri", ignore_code_extraction_warnings=ignore_code_extraction_warnings
-            )
-            LOG.debug("Found Serverless function with name='%s' and CodeUri='%s'", name, codeuri)
+            if resource_properties.get("InlineCode") is not None:
```

Comment:
nit: I don't think we need to explicitly compare to `is not None` here

```suggestion
            if resource_properties.get("InlineCode"):
```